types/aws-lambda/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for AWS Lambda 8.10
+// Type definitions for AWS Lambda 8.11
```

Comment:
Don't change this, the version number is to match the runtime version it's implementing, e.g. the parameters to the handler.